packages/java/tests/spring/endpoints-latest-java/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@polymer/polymer": "3.5.1",
     "@vaadin/common-frontend": "0.0.19",
+    "@vaadin/hilla-file-router": "24.4.0-alpha7",
```

Comment:
Shouldn't this also be a link instead of a solid version?

Reply:
→ https://github.com/vaadin/hilla/pull/2154